src/plugin-manager.js

```diff
@@ -8,7 +8,7 @@ import {FrameworkConfiguration} from 'aurelia-framework';
  * @export
  * @interface PluginDefinition
  */
-export interface PluginDefinition {
+interface PluginDefinition {
```

Comment:
It seems to me typescript export all interfaces anyway, cause it exported on the definition.
But i am not too sure about that